scripts/functional-tests.sh

```diff
@@ -374,10 +374,37 @@ test_model_print(){
 
 }
 
+test_config_value_isset() {
+    VALUE=$1
+    SECTION=$2
+    PROPERTY=$3
+    CONFIG_FILE=$4
+    echo "Check configuration value '$VALUE' in property '$PROPERTY' of section '$SECTION' of '$CONFIG_FILE'"
+
+    PROPERTY_VALUE=$(yq ".$SECTION.$PROPERTY" "$CONFIG_FILE" | xargs)
+    if [[ $PROPERTY_VALUE != *"$VALUE"* ]]; then
+      echo "'$VALUE' not set in property '$PROPERTY_VALUE' of the '$SECTION' section of configuration '$CONFIG_FILE'"
+      exit 1
+    else
+      echo "Value '$VALUE' is set"
+    fi
+}
+
+test_config_model_defaults(){
```

Comment:
I just realized that now, sorry, but these seem more like unit tests than functional tests, in my opinion. If you feel re-writing I think it'd be better, otherwise I won't block for this.

Reply:
I think this makes more sense. Thanks @leseb for the suggestion. Tests now moved to unit tests.